### PR TITLE
Change chains pagination max_limit to 20

### PR DESCRIPTION
**Pull request by fmrsabino**
- Change Chains pagination limit from `100` to `20`

### CODE REVIEW

File: src/chains/views.py [hunk 1]
```diff
@@ -14,7 +14,7 @@
 class ChainsListView(ListAPIView):
     serializer_class = ChainSerializer
     pagination_class = LimitOffsetPagination
-    pagination_class.max_limit = 100
+    pagination_class.max_limit = 20
```
Comment (Uxio0) [inline src/chains/views.py:17]: Isn't this changing the class attributes for every view using it?

Comment (Uxio0) [reply src/chains/views.py:17]: So I guess every view that has the pagination class instantiated after this will have `max_limit=20`

Comment (fmrsabino) [reply src/chains/views.py:17]: Yes you are right. I think it's better to remove this for now and add it to the Django configuration side 🤦

Comment (Uxio0) [reply src/chains/views.py:17]: I wouldn't do that there. I would define a custom LimitOffsetPagination and set the values there

Comment (fmrsabino) [reply src/chains/views.py]: Done a72f7f1

**Review by Uxio0 — APPROVED** [review]

**Pull request merged**